www/front_src/src/Navigation/Sidebar/sideBarAtoms.ts

```diff
@@ -0,0 +1,19 @@
+import { atomWithStorage } from 'jotai/utils';
+
+import { Page } from '../models';
+
+export const selectedNavigationItemsAtom = atomWithStorage<Record<
```

Comment:
You can add another `atom` for hovered items

Reply:
@Thebarda I think it's better to get rid of the hovered items atom and keep only the sleectedNavigationItems one as they basically reflect the same concept

Reply:
Bu we should rid of properties that end with _Navigated, right ?

Reply:
I will make an update to have the same concept, just have the navigated selected items , and the itemsHovered by default atom will be removed